### PR TITLE
add new section

### PR DESCRIPTION
**Pull request by hakuturu583**
Signed-off-by: MasayaKataoka <ms.kataoka@gmail.com>

## Types of PR

- [x] New Features
- [ ] Upgrade of existing features
- [ ] Bugfix

## Description

add documentation about vehicle model.

### CODE REVIEW

**Review by yamacir-kit — APPROVED** [review]

File: docs/design/About.md [hunk 1]
```diff
@@ -31,3 +31,7 @@ If you want to know whole system architecture of this framework, please read [th
 ## TierIV Scenario Format Version 2.0
 
 If you want to know TierIV Scenario format Version 2.0 , which is a default scenario format of this framework, please read [this documentation](TierIVScenarioFormatVersion2.md).
+
+## Vehicle Model
+
+If your want to know about vehicle dynamics model in simulation, please check [this documentation](VehicleDynamics.md).
```
Comment (kmiya) [inline docs/design/About.md:37]:
```suggestion
If you want to know about vehicle dynamics model in simulation, please check [this documentation](VehicleDynamics.md).
```

Comment (hakuturu583) [reply docs/design/About.md]: Tank you!
I fixed at https://github.com/tier4/scenario_simulator_v2/pull/540/commits/5a14e79106e83816def95bdecd4892dcdd08e81d

File: docs/design/VehicleDynamics.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Vehicle Dynamics
+
+In this section, we describe the vehicle model which was implemented in scenario_smulator_v2
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:3]:
```suggestion
In this section, we describe the vehicle model which is implemented in scenario_simulator_v2.
```

File: docs/design/VehicleDynamics.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+## Vehicle Model Types
+
+Pose is calculated by bicycle kinematics model.
+
+Steering and velocity models are implemented 3 types of models.
+
+### IDEAL_STEER
+Uses velocity command. The steering and velocity varies ideally as commanded.
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:13]:
```suggestion
### IDEAL_STEER

Uses velocity command. The steering and velocity vary ideally as commanded.
```

File: docs/design/VehicleDynamics.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+### DELAY_STEER 
+Uses velocity command. The steering and velocity varies following a 1st-order delay model.
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:16]:
```suggestion
### DELAY_STEER

Uses velocity command. The steering and velocity vary following a first-order delay model.
```

File: docs/design/VehicleDynamics.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+### DELAY_STEER_ACC
+Uses acceleration command. The steering and acceleration varies following a 1st-order delay model.
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:19]:
```suggestion
### DELAY_STEER_ACC

Uses acceleration command. The steering and acceleration vary following a first-order delay model.
```

File: docs/design/VehicleDynamics.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+| vel_lim              | double | limit of velocity                                    | x           | o           | o                  | 50.0          | [m/s]   |
+| accel_rate           | double | limit of acceleration                                | x           | o           | o                  | 7.0           | [m/ss]  |
+| steer_lim            | double | limit of steering angle                              | x           | o           | o                  | 1.0           | [rad]   |
+| steer_rate_lim       | double | limit of steering angle change rate                  | x           | o           | o                  | 5.0           | [rad/s] |
+| deadzone_delta_steer | double | dead zone for the steering dynamics                  | x           | o           | o                  | 0.0           | [rad]   |
+
+_Note_: The steering/velocity/acceleration dynamics is modeled by a first order system with a deadtime in a _delay_ model. The definition of the _time constant_ is the time it takes for the step response to rise up to 63% of its final value. The _deadtime_ is a delay in the response to a control input.
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:36]:
```suggestion
_Note_: The steering/velocity/acceleration dynamics is modeled by a first-order system with a deadtime in a _delay_ model. The definition of the _time constant_ is the time it takes for the step response to rise up to 63% of its final value. The _deadtime_ is a delay in the response to a control input.
```

File: docs/design/VehicleDynamics.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+tread_length: 1.0
+
+vehicle_model_type: DELAY_STEER_ACC
+vel_lim: 50.0
+vel_noise_stddev: 0.0
+vel_time_constant: 0.61
+vel_time_delay: 0.25
+```
+
+This example is DELAY_STEER_ACC model, if you want to use another type, please set another variable.
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:68]:
```suggestion
This example shows DELAY_STEER_ACC model. If you want to use another type, please set another variable.
```

File: mkdocs.yml [hunk 8]
```diff
@@ -76,6 +76,7 @@ nav:
       - design/TrafficSimulator.md
       - design/ZeroMQ.md
       - design/Metrics.md
+      - design/VehicleDynamics.md
```
Comment (kmiya) [inline mkdocs.yml:79]: The index of the design/TierIVScenarioFormatVersion2.md is missing.

Comment (hakuturu583) [reply mkdocs.yml:79]: Thank you!
I add in https://github.com/tier4/scenario_simulator_v2/pull/540/commits/a8f6014bc4b86dc9272b9b6d93195be0765696e9

File: docs/design/VehicleDynamics.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+### DELAY_STEER
+
+Uses velocity command. The steering and velocity vary following a first-order delay model.
+
+### DELAY_STEER_ACC
+
+Uses acceleration command. The steering and acceleration vary following a first-order delay model.
+
+## Vehicle Model Parameters
+| Name                 | Type   | Description                                          | IDEAL STEER | DELAY STEER | DELAY STEER 　 ACC | Default value | unit    |
```
Comment (kmiya) [inline docs/design/VehicleDynamics.md:25]:
```suggestion
| Name                 | Type   | Description                                          | IDEAL_STEER | DELAY_STEER | DELAY_STEER_ACC | Default value | unit    |
```
Is it intentional? (Warning: This suggestion might break the column width)

Comment (hakuturu583) [reply docs/design/VehicleDynamics.md]: Thank you.
I fiexed.

Comment (hakuturu583) [reply docs/design/VehicleDynamics.md]: Thank you I fiexed in https://github.com/tier4/scenario_simulator_v2/pull/540/commits/a3c0c2ed4ebabfad7c80aed6bf46f0dbc00d9397

**Review by kmiya — APPROVED** [review]
LGTM :+1:

**Pull request merged**